core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java

```diff
@@ -143,10 +147,17 @@ private void reportResults(IProcessingReporter reporter, Set<IEvaluationResult>
             results.forEach(r ->
                     r.getFailedArtifacts().forEach(a ->
                             reporter.add(MessageType.RULE_ENGINE,
-                                    r.getSeverity() + ": " + r.getDescription())));
+                                    r.getId() + " (" + r.getSeverity() + "): " + getCoordinates(a) + " : " + r.getDescription())));
```

Comment:
Can you sort the `RULE_ENGINE` report entry by the IEvaluationResult ID? It's better for readability.

Reply:
Good point, should be done by now.